Add network route (#83)
harvester/docs: docs/networking/harvester-network.md

### Enabling Default VLAN Network

> Users can enable the VLAN network via **Setting > VLAN** and selecting a network interface for the nodes as the default VLAN config .

Reviewer: ```suggestion
You can enable VLAN network via **Setting > VLAN**. Once enabled, you will be able to select a network interface for the nodes as the default VLAN config.
```

> It is recommended to choose a separate network interface for the VLAN other than the one used for the management network (i.e., `harvester-mgmt`) for better network performance and isolation.

Reviewer: ```suggestion
For better network performances and isolation, we recommend to choose a separate network interface for the VLAN other than the one used for the management network (i.e., `harvester-mgmt`).
```

A new VLAN network can be created via the **Advanced > Networks** page and clicking the **Create** button.

 - Specify the name and VLAN ID that you want to create for the VLAN network.
   ![create-vlan-network.png](assets/create-network.png)
  

>  - Configure the route so that users can connect the IPv4 address in the VLAN network from the hosts. The CIDR and gateway of the VLAN network are necessary parameters of the route. There are two methods to get them.

Reviewer: ```suggestion
 - Configure a route in order to allow the hosts to connect to the VLAN network using IPv4 addresses. The CIDR and gateway of the VLAN network are mandatory parameters for the route configuration.  You can configure the route by choosing one of the following options:
```

>    - auto(DHCP) mode: Harvester network controller will get the CIDR and gateway through DHCP protocol if there is a DHCP server. Moreover, users can specify the DHCP server address.

Reviewer: ```suggestion
   - auto(DHCP) mode: the Harvester network controller will get the CIDR and gateway values from the DHCP server using the DHCP protocol. You need to specify the DHCP server address.
```

Author: It's optional to specify the DHCP server address.

Reviewer: rephrased based on the feedback

   ![](assets/create-network-auto.png)

>    - manual mode: Users should provide CIDR and gateway manually.

Reviewer: ```suggestion
   - manual mode: You need to specify the CIDR and gateway values manually.
```
